capstone-ui/src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./SearchPage.css";
+import MentorCard from "./MentorCard/MentorCard";
+import Menu from "./Menu/Menu";
+import { useState, useEffect } from "react";
+
+export default function SearchPage() {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [mentors, setMentors] = useState([]);
+
+  useEffect(() => {
+    fetch("https://api-staging.codingcoach.io/mentors")
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          setIsLoaded(true);
+          setMentors(result);
+        },
+        (error) => {
+          setIsLoaded(true);
+          setError(error);
+        }
+      );
+  }, []);
+  console.log("data:", mentors.data);
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  } else if (!isLoaded) {
+    return <div>Loading...</div>;
+  } else {
```

Comment:
Since you have several returns, you can avoid ELSE and ELSE IF,
```
if (error) {
  return <div>Error: {error.message}</div>;
}

else if (!isLoaded) {
  return <div>Loading...</div>;
}

return (
  <div className="main">
  ...
)
```

Try to avoid ELSE, and mostly ELSE IF.